src/HttpClient/HttpClientFactory.php

```diff
@@ -111,7 +113,20 @@ public function create(Options $options): HttpAsyncClientInterface
         }
 
         if (null === $httpClient) {
-            if (class_exists(GuzzleHttpClient::class)) {
+            if (class_exists(SymfonyHttplugClient::class)) {
+                $symfonyConfig = [
+                    'timeout' => self::DEFAULT_HTTP_TIMEOUT,
```

Comment:
Yeah it wasn't exactly clear to me, but I think you are right.

Do you think we should combine our 2 timeouts into that one since there is no separate connect timeout or should we just leave the connect timeout off here?

Reply:
I think that going for the second option is fine in this case, as I have no idea if the `max_duration` already takes into account the connection phase or not

Reply:
Agreed, updated.